JoyboyCommunity/assets/svgs/svgComponents/Exit.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from "react"
+import Svg, { SvgProps, Rect, Path } from "react-native-svg"
+const Exit = (props: SvgProps) => (
+  <Svg
+    width={57}
+    height={56}
+    fill="none"
+    {...props}
+  >
+    <Rect
+      width={56}
+      height={56}
+      x={0.5}
+      fill="#EC796B"
+      fillOpacity={0.1}
```

Comment:
colors should be currentColor, and the usage should include the color, not the definition. For example, this should be `fill="currentColor"` and usage would be `<ExitIcon color="#EC796B" />`